lib/knexFunctions.js

```diff
@@ -14,19 +15,133 @@ function createGame(id){
   })
```

Comment:
So this is still hard-coded, right? Is there any way we can get this to be dynamic or is it more complex than that? 😕